spec/controllers/games_controller_spec.rb

```diff
@@ -7,4 +7,13 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'games#create action' do
+    it 'should successfully create a new game in our database' do
+      post :create, params: { game: { user_id_black: 15 } }
+      @game = Game.last
+      expect(response).to redirect_to game_path[:@game.id]
```

Comment:
This line should use parenthesis for the `game_path` method call, rather than square brackets - you can also let the path helper figure out the ID by just passing it `@game` rather than `@game.id`:

```ruby
expect(response).to redirect_to game_path(@game)
```

I _think_ that should fix the issue you're seeing from Rspec